spectacles/exceptions.py

```diff
@@ -16,13 +16,15 @@ def __init__(
         path: str,
         message: str,
         sql: str,
+        query_url: str,
```

Comment:
Are you okay with calling this `explore_url` for clarity?

Reply:
Yeah 👍 I'll change that.